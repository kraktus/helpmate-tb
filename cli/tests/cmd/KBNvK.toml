args = "--no-time generate -m \"KBNvK\" --tb-dir ../table"
bin.name = "helpmate_tb_cli"
