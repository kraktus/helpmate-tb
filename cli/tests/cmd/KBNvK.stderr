dhat: Total:     81,[..],[..] bytes in [..] blocks
dhat: At t-gmax: 12,[..],[..] bytes in [..] blocks
dhat: At t-end:  [..] bytes in [..] blocks
dhat: The data has been saved to dhat-heap.json, and is viewable with dhat/dh_view.html
